app/src/js/utils/table-config/reconciliation-reports.js

```diff
@@ -335,6 +340,8 @@ export const tableColumnsGranuleConflictDetails = ({ reportType }) => {
       Header: 'In Cumulus Only',
       id: 'onlyInCumulus',
       accessor: 'reason',
+      // TODO: Updated ES lint rules caused a changeset-unrelated violation - this should be fixed
+      // eslint-disable-next-line react/prop-types
```

Comment:
Do we need a ticket? I guess if it's disabled and only in these couple spots it's not a huge deal. It can be addressed the next time something touches these lines.

Reply:
Probably, but there's a lot to unpack in that question.     Given we're *already* disabling this rule, I'm not inclined to do so directly here, given I don't know the history behind the decision to ignore it elsewhere in the codebase.         The only reason this came up was the vuln update caused me to update eslint and they fixed a bug that wasn't catching this.